CreeDictionary/CreeDictionary/templates/CreeDictionary/word-entries.html

```diff
@@ -19,6 +20,14 @@ <h2 class="definition-title definition-title--search-result">
                             <span lang="cr">{{ result.matched_cree }}</span>
                         {% endif %}
                         <span class="definition-title__pos">({{ result.lemma_wordform|presentational_pos }})</span>
+                        {% if result.linguistic_breakdown_head or result.linguistic_breakdown_tail %}
+                        <div class="definition-title__tooltip">
```

Comment:
Note to @kobexamoh: this could be implemented using:

```html
<details>
   <summary role="button"><img alt="linguistic breakdown" aria-label="Expand linguistic summary" href="..."></summary>
   <aside>
     [linguistic breakdown]
   </aside>
</details>
```

Reply:
wow so much more semantic

Reply:
and accessible